.github/workflows/test_build_wheels_windows_without_cuda.yml

```diff
@@ -0,0 +1,62 @@
+name: Test Build Windows Wheels without CUDA
+
+on:
+  pull_request:
+    paths:
+      - .github/actions/setup-binary-builds/action.yml
+      - .github/workflows/test_build_wheels_windows_without_cuda.yml
+      - .github/workflows/build_wheels_windows.yml
+      - .github/workflows/generate_binary_build_matrix.yml
+      - tools/scripts/generate_binary_build_matrix.py
+  workflow_dispatch:
+
+jobs:
+  generate-matrix:
+    uses: ./.github/workflows/generate_binary_build_matrix.yml
+    with:
+      package-type: wheel
+      os: windows
+      test-infra-repository: ${{ github.repository }}
+      test-infra-ref: ${{ github.ref }}
+      with-cuda: disable
+  test:
+    needs: generate-matrix
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - repository: pytorch/audio
+            pre-script: ""
+            env-script: packaging/vc_env_helper.bat
+            wheel-build-params: "--plat-name win_amd64"
+            post-script: ""
+            smoke-test-script: ""
+            package-name: torchaudio
+          # - repository: pytorch/vision
```

Comment:
Please cleanup these if not used

Reply:
I'm currently working on enabling torchvision. This PR is a pre-requisite.